man/systemd.netdev.xml

```diff
@@ -310,6 +310,15 @@
             of the Listening and Learning states before the Forwarding state is entered.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>MulticastQuerier=</varname></term>
+          <listitem>
+            <para>A boolean. MulticastQuerier specifies process of listening to Internet Group Management Protocol (IGMP)
+            network traffic. The feature allows a network bridge to listen in on the IGMP conversation between hosts and routers.
+            Defaults to false.
```

Comment:
A boolean. This setting controls the `IFLA_BR_MCAST_QUERIER` option in the kernel. If enabled, the kernel will send general ICMP queries from a zero source address. This feature should allow faster convergence on startup, but it causes some multicast-aware switches to misbehave and disrupt forwarding of multicast packets. If not set, the kernel default (currently "no"), will be preserved.